swap_meet/item.py

```diff
@@ -1,2 +1,69 @@
 class Item:
-    pass
\ No newline at end of file
+    """
+    A class to represent an item.
+
+    ...
+    Attributes:
+
+    condition : int or float, optional
+        attribute describing the condition of the item, defaulting to 0. higher numbers indicate better condition.
+    age: int or float, optional
+        attribute describing how many years old an item is, defaulting to None. cannot be negative.
+    category: str, optional
+        attribute to describe the kind of item, defaulting to an empty string.
+    
+    ...
+    Methods:
+
+    __str__:
+        represents class objects as a string.
+    
+    condition_description():
+        Returns string describing the condition of the item
+    
+    age_description():
+        Returns string describing item's age
+
+    """
+    
+    def __init__(self, condition = 0, age = None, category = ""):
+        self.condition = condition
+        self.category = category
+        if age and age < 0:
+            raise ValueError("Age may not be a negative number")
+        self.age = age
+
+
+    def __str__(self):
+        return "Hello World!"
+    
+    def condition_description(self):
+        """Returns string describing the condition of the item"""
+
+        if self.condition <= 1:
+            return "I mean I wouldn't touch this with a ten foot pole but, uh, you do you."
+        elif self.condition <= 2:
+            return "It's....tolerable. I suppose. If one must."
+        elif self.condition <= 3:
+            return "Decent."
+        elif self.condition <= 4:
+            return "Alright, NOW we're talking. Nothing's perfect, right?"
+        elif self.condition <= 5:
+            return "Excellent. Full marks. Drinks all around!"
+        else:
+            return "Oh my god BUY THIS NOW ITS CONDITION IS OUT OF THIS WORLD"
```

Comment:
Ah, I think I had missed or forgot that condition would necessarily always be between 0 and 5, and so was trying to account for the edge case that someone put in a higher or lower number, even if typical usage was to have condition in that range. 

In the condition map situation, if someone puts a float in, does the int() round the condition up or down (say, 4.2 resolves to 4 but 4.6 resolves to 5), or will it just resolve to the integer portion of the float (both 4.2 and 4.6 resolving to 4)?

Reply:
`int` will do the latter, it will chop off the decimals and just give you the integer. So the solution above works if you want everything from 4.0-4.99 to have the same text. If you wanted to return the text for whichever whole number the decimal is closest to, the `round()` function would do that.